sys/linux/socket_netlink_generic_devlink.txt

```diff
@@ -71,3 +71,103 @@ devlink_port_set {
 	port_handle		devlink_port_handle
 	DEVLINK_ATTR_PORT_TYPE	nlattr[DEVLINK_ATTR_PORT_TYPE, int16[DEVLINK_PORT_TYPE_NOTSET:DEVLINK_PORT_TYPE_IB]]
 } [packed, align_4]
+
+type msghdr_nl_devlink_port_split msghdr_netlink[netlink_msg_t[genl_devlink_family_id, genlmsghdr_t[DEVLINK_CMD_PORT_SPLIT], devlink_port_split]]
+sendmsg$DEVLINK_CMD_PORT_SPLIT(fd sock_nl_generic, msg ptr[in, msghdr_nl_devlink_port_split], f flags[send_flags])
+
+devlink_port_split {
+	port_handle			devlink_port_handle
+	DEVLINK_ATTR_PORT_SPLIT_COUNT	nlattr[DEVLINK_ATTR_PORT_SPLIT_COUNT, int32[0:8]]
+} [packed, align_4]
```

Comment:
`align_4` syntax was changed to `align[4]`
But I think these are not required because each nlattr member has own alignment, so adding alignment to the whole struct is not required.